runtime/onert/backend/cpu/ops/MaxLayer.cc

```diff
@@ -29,21 +29,25 @@ namespace cpu
 namespace ops
 {
 
-void MaxLayer::maxFloat32()
+template <typename T> void MaxLayer::maximum()
 {
-  nnfw::cker::Max<float>(getTensorShape(_lhs), reinterpret_cast<const float *>(_lhs->buffer()),
-                         getTensorShape(_rhs), reinterpret_cast<const float *>(_rhs->buffer()),
-                         getTensorShape(_output), reinterpret_cast<float *>(_output->buffer()));
+  nnfw::cker::Max<T>(getTensorShape(_lhs), reinterpret_cast<const T *>(_lhs->buffer()),
+                     getTensorShape(_rhs), reinterpret_cast<const T *>(_rhs->buffer()),
+                     getTensorShape(_output), reinterpret_cast<T *>(_output->buffer()));
 }
 
 void MaxLayer::maxQuant8()
 {
-  // TODO Check whether cker for quant8 max produces correct results
-  // nnfw::cker::Max<uint8_t>(
-  //     getTensorShape(_lhs), reinterpret_cast<const uint8_t*>(_lhs->buffer()),
-  //     getTensorShape(_rhs), reinterpret_cast<const uint8_t*>(_rhs->buffer()),
-  //     getTensorShape(_output), reinterpret_cast<uint8_t*>(_output->buffer()));
-
+  if (_lhs->data_scale() == _rhs->data_scale() && _lhs->data_scale() == _output->data_scale())
+  {
+    if (_lhs->data_offset() == _rhs->data_offset() && _lhs->data_offset() == _output->data_offset())
+    {
+      return nnfw::cker::Max<uint8_t>(
```

Comment:
Please remove `return`

Reply:
If you remove **return**, line 51 is always called.

So I think **return** is necessary

Reply:
Ah I see. I forgot it.